src/components/IconSwitch/IconSwitch.story.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import List16 from '@carbon/icons-react/lib/list/16';
+import List20 from '@carbon/icons-react/lib/list/20';
+import List24 from '@carbon/icons-react/lib/list/24';
```

Comment:
Carbon provides a central entrypoint now, so the paths don't need to be fully qualified.

```suggestion
import List16 from '@carbon/icons-react';
import List20 from '@carbon/icons-react';
import List24 from '@carbon/icons-react';
```

Reply:
Ooooh much cleaner - done!